# HoCVector.msg
Header header
int32 id
float32[] hue_vector
float32[] sat_vector
